srcpkgs/kubernetes-helm/template

```diff
@@ -35,23 +35,10 @@ do_extract() {
 		helm-${version}
 }
 
-pre_build() {
-	export GOPATH="$(pwd)/golib"
-
-	## The current version of glide in the repository has
-	## a bug that prevents building, so pull the current
-	## version from git.
-	CC=/bin/gcc go get github.com/Masterminds/glide
-	CC=/bin/gcc go get github.com/mitchellh/gox
-	CC=/bin/gcc go install github.com/Masterminds/glide
-	CC=/bin/gcc go install github.com/mitchellh/gox
-	PATH="$PATH:$(pwd)/golib/bin/"
-	export PATH
-}
-
```

Comment:
Did they completely stop needing glide? Or does the build system now build it?

Reply:
Yes, glide seems to have gone from the dependencies. I have built the package and been using it locally and it is working fine.